src/components/MoveReference/index.tsx

```diff
@@ -198,7 +199,7 @@ const Content = ({ branch, page }: ContentProps) => {
         <ReactMarkdown
           children={content}
           rehypePlugins={[rehypeRaw]}
-          remarkPlugins={[remarkGfm]}
+          remarkPlugins={[remarkGfm, remarkMermaid as any]}
```

Comment:
Awesome!  Guess it doesn't match the type correctly?

Reply:
yes... the type doesn't matched.